the-experts-BE/app.js

```diff
@@ -6,12 +6,16 @@ var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
+var testAPIRouter = require("./routes/testAPI");
+var cors = require("cors");
 
 var app = express();
 
 // view engine setup
-// app.set('views', path.join(__dirname, 'views'));
-// app.set('view engine', 'jade');
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'jade');
```

Comment:
we just went over this that we don't need it, and made it work without it. but i guess for the sake of progression we will keep it for now

Reply:
cool